lesson_00/chigazograham/README.md

```diff
@@ -0,0 +1,31 @@
+# Chigazo Grahams READ.ME
+
+## Who is Chigazo?
+    My full name is Chigazo Austin Graham I was born and raised here, in Wilmington, Delaware. Growing up I was a part of various programs teaching me skills in business and technology, I even owned my own business selling hygiene products by the age of 7 and won at a business pitch competition against high school students while in middle school. I graduated from Salesianum high school and during my time there I was shortly a part of the Code Differently Youth Program as a junior software engineer before departing to focus on my studies. I attended Delaware State University (DSU) with a major in computer science for a year prior to joining the Code Differently adult program to further my current software engineering skills and to begin building a career for myself. 
```

Comment:
It's weird to have to scroll to see this text and I'm not sure why that's happening.